### ISSUE TITLE
Tailwind.Traders.Profile.Api.GetAllProfiles : check for result == null ?

### ISSUE THREAD

**Issue by StefH** [microsoft/TailwindTraders-Backend#32]
The current code looks like:
``` c#
var result = await _ctx.Profiles
    .Select(p => p.ToProfileDto(_settings))
    .ToListAsync();

if(result == null)
{
    return NoContent();
}

return Ok(result);
```

However, can `result` ever be null ? In case nothing is found in database, an empty list is returned?

**Comment by ivilches** [comment]
You're right @StefH! Solved in #38 
Thank you!

**Comment by olgamarti** [comment]
Solved in commit c5b93d4
